doc/lua_api.txt

```diff
@@ -4125,6 +4125,9 @@ Definition tables
     --  ^ Limit automatic rotation to this value in degrees per second,
     --    value < 0 no limit.
         backface_culling = true, -- false to disable backface_culling for model
+        glow = 0,
+    --  ^ Add this much extra lighting when calculating texture color.
+          value < 0 disables light's effect on texture color
```

Comment:
Maybe add a very short explanation of why this is useful and what it allows? (mod setting of color or whatever).